README.md

```diff
@@ -1 +1,97 @@
-# case-statements-quiz
+#Case Statements Quiz
+## Objectives
+
+1. Distinguish a case statement from other patterns
+2. Identify when to use a case statement
+3. Write a case statement
+
+???
+
+# Quiz
+
+?: Which of the following is a case statement:
+
+A. 
+
+```ruby
+name = "Steven"
+
+if name == "Steven"
+  "Hi, #{name}"
+else
+  puts "Hi, stranger!"
+``` 
+
+B.
+
+```ruby
+name = "Steven"
+
+case name
+  when "Steven"
+    puts "Hi, #{name}"
+  when "Amanda"
+    puts "Welcome back, #{name}"
+  when "Admin"
+    puts "You have all the power!"
+  end
+``` 
+
+C.
+
+```ruby
+def case
+  puts "Am I a case statement?"
+end
+
+case
+```
+
+( ) A
+(X) B
+( ) C
+
+
+?: Of the following two examples, which is best case for a case statement:
```

Comment:
"Best case for a case statement" don't use case the first time around, confusing. maybe "which example uses the case statment the best?"